endpoints.html

```diff
@@ -6,7 +6,10 @@
 <div class="home">
   <p>You can make HTTP GET, POST or PUT Requests. Requests are passed to the API by using JSON objects to the API endpoints with the appropriate parameters. The documentation for each API call will contain more detail on the parameters accepted by the call.</p>
   <p>All responses are sent JSON-encoded. The specific responses (successful ones) are described in the documentation section for each method. See <a href="https://en.wikipedia.org/wiki/Json">https://en.wikipedia.org/wiki/Json</a> for more information on JSON.</p>
-  <p>All input strings should be UTF-8 encoded, our response is always UTF-8 encoded.</p>
+  <p>
+    All input strings should be UTF-8 encoded, our response is always UTF-8 encoded.
+    Optional JSON properties can be omitted from the request, null values are only allowed for strings.
```

Comment:
This is incorrect, null values are not 'allowed' for strings. There is nothing in the specifications saying you are allowed to do so. Even so, you are _not_ allowed. It is shortcoming on our part that this is currently possible but this is not by design.